src/components/Scan/GenericQRScanDialog.tsx

```
@@ -0,0 +1,211 @@
+                    <span className="bg-white px-2 text-sm text-gray-500">
+                      {t("or")}
+                    </span>
+                  </div>
+                </div>
+              </div>
+
+              <div className="space-y-4">
+                <div>
+                  <label className="text-sm sm:text-base font-medium text-gray-700">
+                    {labelText}:
+                  </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)